pkg/instancegroups/instancegroups.go

```diff
@@ -221,6 +233,64 @@ func (r *RollingUpdateInstanceGroup) RollingUpdate(rollingUpdateData *RollingUpd
 	return nil
 }
 
+func (r *RollingUpdateInstanceGroup) taintAllNeedUpdate(update []*cloudinstances.CloudInstanceGroupMember, rollingUpdateData *RollingUpdateCluster) error {
```

Comment:
pretty much the only thing I don't like is the naming here. I would prefer something like `taintOutdatedNodes`, but i'm not a hard no, it just feels awkward to me.